src/utils/dig.js

```diff
@@ -72,12 +72,12 @@ export const handleDig = async ({ domain, types, short, provider }) => {
     };
 
     // Convert results to an embed
-    return results.map(({ type, data }) => createEmbed(`${type} records`, present(type, data), 'diggy diggy hole'));
+    return results.map(({ type, data }) => createEmbed(`${type} records`, present(type, data), 'diggy diggy hole :warning: cd bit set for request, DNSSEC validation disabled'));
```

Comment:
As I noted in the original issue, we should only be adding the warning if the cdflag is actively being set. Showing this warning all the time would be incorrect and misleading

Reply:
Alright!